bootstrap.sh

```diff
@@ -14,7 +14,7 @@ if test -f unit_test.sh; then
 fi
 
 # Create tmp dir to store data in during job run (do NOT store in $WORKSPACE)
-export TMP_JOB_DIR=$(mktemp -d -p "$HOME" -t "jenkins-${JOB_NAME}-${BUILD_NUMBER}-XXXXXX")
+export TMP_JOB_DIR=${TMP_JOB_DIR:-$(mktemp -d -p "$HOME" -t "jenkins-${JOB_NAME}-${BUILD_NUMBER}-XXXXXX")}
```

Comment:
Do we want to allow overwriting this variable? I'm thinking someone adding "TMP_JOB_DIR=$WORDKIR" , `$PWD` or similar would defeat the entire "tempdir outside of workdir" approach

Reply:
OK I think I understand the need for allowing to rewrite the TMP dir - if you can review RedHatInsights/bonfire#361 and think it may work , this should not be required anymore, please LMK your thoughts